packages/editor/src/bindings/post-meta.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { select } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../store';
+
+const { getCurrentPostType } = select( editorStore );
+
+// Prettify the name until the label is available in the REST API endpoint.
+const keyToLabel = ( key ) => {
+	return key
+		.split( '_' )
+		.map( ( word ) => word.charAt( 0 ).toUpperCase() + word.slice( 1 ) )
+		.join( ' ' );
+};
+
+export default {
+	name: 'post_meta',
+	label: 'Post Meta',
+	component: null,
```

Comment:
What’s the role of this setting? It doesn’t seem to be needed at the moment.

Reply:
It will potentially be used in the future when we have the UI to create the bindings. Should I remove it until then?

Reply:
Yes, let’s keep only usable code.

Reply:
Done in [this commit](https://github.com/WordPress/gutenberg/pull/58085/commits/1984749af6502fe017903c15f6f5f6b05c21ae8c).